src/App.js

```diff
@@ -69,6 +70,9 @@ export default function App() {
         <Route path="/success">
           <SuccessPage />
         </Route>
+        <Route path="/shift-signup">
+          <ShiftSignUp toggleLoggedIn={toggleLoggedIn} />
```

Comment:
<PrivateRoute isLoggedIn={isLoggedIn} path="/shift-signup">
          <ShiftSignUp  />
        </PrivateRoute>
        
The route should be called like this,